test/test_helper.exs

```diff
@@ -1,3 +1,48 @@
 Code.require_file "support/test_repo.exs", __DIR__
+
+# For mix tests
+Mix.shell(Mix.Shell.Process)
+
 ExUnit.start()
 Application.ensure_all_started(:bypass)
+
+defmodule MixHelper do
```

Comment:
@tiagoengel is that possible to move this module to a file in `/support` folder?